libraries/chainbase/include/chainbase/chainbase.hpp

```diff
@@ -467,6 +505,13 @@ namespace chainbase {
             _revision = revision;
          }
 
+         void remove_object( int64_t id )
```

Comment:
Why is this being added back in?

Reply:
@mvandeberg Accidentally added back. Removed in last commit 217e200993a70858e9f10358c173ace10cf3f256